app/models/recording.rb

```diff
@@ -2,13 +2,14 @@
 
 class Recording < ApplicationRecord
   belongs_to :room
+  has_one :user, through: :room
```

Comment:
Because this association was added, rails was getting confused between `users` and `user` so I renamed `users` to `participants`